source/tests/system/nisync_driver_api_tests.cpp

```diff
@@ -984,6 +1065,159 @@ TEST_F(NiSyncDriverApiTest, DISABLED_SetTimeReference8021AS_ReturnsSuccess)
   EXPECT_EQ(VI_SUCCESS, viStatus);
 }
 
+TEST_F(NiSyncDriverApiTest, DISABLED_CreateClearFutureTimeEvent_ReturnsSuccess)
+{
+  ViStatus viStatus;
+
+  ViUInt32 timeSeconds, timeNanoseconds;
+  ViUInt16 timeFractionalNanoseconds;
+  auto grpcStatus = call_GetTime(
+     &timeSeconds,
+     &timeNanoseconds,
+     &timeFractionalNanoseconds, // ignored
+     &viStatus);
+  ASSERT_TRUE(grpcStatus.ok());
+
+  timeSeconds += 30;
+  grpcStatus = call_CreateFutureTimeEvent(
+     NISYNC_VAL_PFI1, // terminalName
+     NISYNC_VAL_LEVEL_LOW, // outputLevel
+     timeSeconds,
+     0, // timeNanoseconds, ignored
+     0, // timeFractionalNanoseconds, ignored
+     &viStatus);
+
+  EXPECT_TRUE(grpcStatus.ok());
+  EXPECT_EQ(VI_SUCCESS, viStatus);
+
+  grpcStatus = call_ClearFutureTimeEvents(
+     NISYNC_VAL_PFI1,
+     &viStatus);
+
+  EXPECT_TRUE(grpcStatus.ok());
+  EXPECT_EQ(VI_SUCCESS, viStatus);
+}
+
+TEST_F(NiSyncDriverApiTest, DISABLED_CreateFutureTimeEventWithInvalidTerminal_ReturnsError)
+{
+  ViStatus viStatus;
+
+  ViUInt32 timeSeconds, timeNanoseconds;
```

Comment:
It might make these tests a little cleaner if we created a helper called `GetCurrentBoardTime` or some other better name.  Calling the get time function takes up a decent chunk of space in the test method, and we only really care about seconds.  You could create the helper I mentioned and just have it return seconds.  It could still assert that status was good as well.

With the helper it'd look something like
```
ViUInt32 timeSeconds = GetBoardTime() + 30;
```